qa/workunits/ceph-helpers.sh

```diff
@@ -1607,6 +1649,19 @@ function test_wait_background() {
     if [ ! -z "$pids" ]; then return 1; fi
 }
 
+function get_admin_socket()
+{
```

Comment:
The cephtool/test.sh is being run as part of make check as well as in the rados suite.  Previously, nobody was trying to find the admin socket in code like that anywhere I found.  The code needs to check for the $CEPH_* environment variables to determine if we are in make check or rados suite.  The rados suite is the one that uses the $dir everywhere but the other does not.  Suggestions?